src/containers/ProfilePage/index.jsx

```diff
@@ -0,0 +1,98 @@
+import React from "react";
+import { Container, Row } from "react-bootstrap";
+import { useForm } from "react-hook-form";
+import UserDetail from "../../components/UserDetail";
+import UserEmail from "../../components/UserEmail";
+import UserPassword from "../../components/UserPassword";
+import defaultProfileImage from "../../images/defaultProfileImage.png";
+import "./index.scss";
+
+function getArrowClass(selectedElm, currentElm) {
+  return selectedElm === currentElm ? "arrow down" : "arrow right";
```

Comment:
elm is not an abbreviation that is common. elm is a tree. el could be acceptable but i'd still just prefer Element.